app/config/parameters.yml.test

```diff
@@ -0,0 +1,23 @@
+parameters:
```

Comment:
you will need to change

https://github.com/tolry/mental-note/blob/b0391a98a945d720cac13fd04c7b6ed1c87316d6/.travis.yml#L41

To make travis use this file, Scrutinizer is configured within scrutinizer, I think - will have to check

Reply:
scrutinizer automatically uses `parameters.dist.yml`, see https://scrutinizer-ci.com/g/tolry/mental-note/inspections/d8c5ac24-1fe8-4334-a5ea-a26616bb512c/log

`sed -i 's/database_host: .*/database_host: 127.0.0.1/' app/config/parameters.yml.dist`

so I would really prefer keeping the dist.yml as template as well as for the tests (travis, scrutinizer) and your setup using a different file or `sed`ing your settings or similar